sidebars.js

```diff
@@ -299,8 +299,42 @@ const sidebars = {
             label: 'opBNB',
             href:'https://docs.bnbchain.org/opbnb-docs/',
           },
-         
-    ],
+
+          {
+              type: 'category',
```

Comment:
The layout seems wrong, after I run the doc site, I dont know where to go to the pages?